cmd/gogio/androidbuild.go

```diff
@@ -467,6 +470,19 @@ func exeAndroid(tmpDir string, tools *androidTools, bi *buildInfo, extraJars, pe
 				<category android:name="android.intent.category.LAUNCHER" />
 			</intent-filter>
 		</activity>
+		{{if .HasService}}
+		<service android:name="org.gioui.GioForegroundService"
+			android:stopWithTask="true">
+			<meta-data android:name="org.gioui.ForegroundChannelID"
+			android:value="ForegroundChannelID" />
+			<meta-data android:name="org.gioui.ForegroundChannelName"
+			android:value="ForegroundChannelName" />
+			<meta-data android:name="org.gioui.ForegroundChannelDesc"
+			android:value="ForegroundChannelDesc" />
```

Comment:
Can the description be optional if there's nothing interesting to put there? Otherwise add a short description.